src/Interfaces/Managers/DocumentManagerInterface.php

```diff
@@ -31,4 +31,14 @@ public function list(
         ?string $expression = null,
         ?array $attributeValues = null
     ): array;
+
+    /**
+     * Update document item.
+     *
+     * @param string $objectId Document object id
+     * @param \LoyaltyCorp\Auditing\Interfaces\DataObjectInterface $dataObject Data transfer object
+     *
+     * @return \Aws\Result
```

Comment:
What happened to making externals?

Reply:
@edwardmurrell-loyaltycorp I asked the same thing in slack - https://loyaltycorp.slack.com/archives/C9Q8WMBJM/p1558678543008000

Reply:
Ah, right. Thanks for linking the relevant conversation, makes it easier.

Reply:
DocumentManager is tightly bound with DynamoDb hence it return AWS result. As this is an external package itself it doesn't have to wrap tightly coupled dependency to make it "external".

It is in not meant to expose to consumer/user but the consuming service should convert into respective response.

For the sake of resolving this issue I've created `Response` object.